Quarkus Artemis dev services fail to start when running with Quarkus snapshot

Quarkus has reorganised some of the dev services config:

https://github.com/quarkusio/quarkus/commit/bbb10098d8fc0b584990f37595bf02a84f5349d5

It makes Quarkus Artemis dev services fail due to:

```
Caused by: java.lang.NoSuchFieldError: timeout
   at io.quarkus.artemis.core.deployment.DevServicesArtemisProcessor.start(DevServicesArtemisProcessor.java:128)
```

blocks
- https://github.com/apache/camel-quarkus/issues/6791

I'm not sure how to fix it in the current codebase. Or we have to fix it when the next Quarkus release?

--- a/core/deployment/src/main/java/io/quarkus/artemis/core/deployment/DevServicesArtemisProcessor.java
+++ b/core/deployment/src/main/java/io/quarkus/artemis/core/deployment/DevServicesArtemisProcessor.java
@@ -28,7 +28,7 @@ import io.quarkus.deployment.builditem.DockerStatusBuildItem;
 import io.quarkus.deployment.builditem.LaunchModeBuildItem;
 import io.quarkus.deployment.console.ConsoleInstalledBuildItem;
 import io.quarkus.deployment.console.StartupLogCompressor;
-import io.quarkus.deployment.dev.devservices.GlobalDevServicesConfig;
+import io.quarkus.deployment.dev.devservices.DevServicesConfig;
 import io.quarkus.deployment.logging.LoggingSetupBuildItem;
 import io.quarkus.devservices.common.ConfigureUtil;
 import io.quarkus.devservices.common.ContainerAddress;
@@ -58,7 +58,7 @@ public class DevServicesArtemisProcessor {
     static volatile boolean first = true;
 
     @SuppressWarnings("unused")
-    @BuildStep(onlyIfNot = IsNormal.class, onlyIf = GlobalDevServicesConfig.Enabled.class)
+    @BuildStep(onlyIfNot = IsNormal.class, onlyIf = DevServicesConfig.Enabled.class)
     public List<DevServicesResultBuildItem> startArtemisDevService(
             DockerStatusBuildItem dockerStatusBuildItem,
             LaunchModeBuildItem launchMode,
@@ -69,7 +69,7 @@ public class DevServicesArtemisProcessor {
             Optional<ConsoleInstalledBuildItem> consoleInstalledBuildItem,
             CuratedApplicationShutdownBuildItem closeBuildItem,
             LoggingSetupBuildItem loggingSetupBuildItem,
-            GlobalDevServicesConfig devServicesConfig) {
+            DevServicesConfig devServicesConfig) {
         ArrayList<DevServicesResultBuildItem> results = new ArrayList<>();
         for (String name : bootstrap.getConfigurationNames()) {
             ArtemisBuildTimeConfig buildTimeConfig = buildConfigs.configs().get(name);
@@ -109,7 +109,7 @@ public class DevServicesArtemisProcessor {
             Optional<ConsoleInstalledBuildItem> consoleInstalledBuildItem,
             CuratedApplicationShutdownBuildItem closeBuildItem,
             LoggingSetupBuildItem loggingSetupBuildItem,
-            GlobalDevServicesConfig devServicesConfig) {
+            DevServicesConfig devServicesConfig) {
         if (devServices.get(name) != null && configuration != null) {
             boolean shouldShutdownTheBroker = !configuration.equals(cfgs.get(name));
             if (!shouldShutdownTheBroker) {
@@ -130,7 +130,7 @@ public class DevServicesArtemisProcessor {
                             dockerStatusBuildItem,
                             configuration,
                             launchMode,
-                            devServicesConfig.timeout);
+                            devServicesConfig.timeout());
                     if (service != null) {
                         devServices.put(name, service);
                     }

